sync/class.jetpack-sync-module-options.php

```diff
@@ -71,16 +71,22 @@ function get_full_sync_actions() {
 	// Is public so that we don't have to store so much data all the options twice.
 	function get_all_options() {
 		$options = array();
+		$random_string = wp_generate_password();
 		foreach ( $this->options_whitelist as $option ) {
-			$options[ $option ] = get_option( $option );
+			$option_value = get_option( $option, $random_string );
```

Comment:
This worries me a bit because it relies on the option value never being set to the string returned by wp_generate_password(), but I suppose the odds of such a collision actually occuring are so low that it's a non issue.